src/api/apicall.js

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+const auth =
+	"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI4MTAyMWNjNC00YjFkLTQyOGItYjJmMC0wNjhkYTQ4YTk4MzQiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.45ynQ6aZhoM1zsNwIKCYR_IATaszKn0ssvnPPQkKL8E";
+const getAllProducts = async () => {
+	try {
+		const response = await axios.get("/api/products");
+		return { products: response.data.products, success: true };
+	} catch (err) {
+		console.log(err);
+		return { products: [], success: false };
+	}
+};
+
+const getWishlist = async () => {
+	const headers = { authorization: auth };
+	try {
+		const response = await axios.get("/api/user/wishlist", { headers });
+		console.log(response);
+		return { wishlist: response.data.wishlist, success: true };
+	} catch (err) {
+		console.log(err);
+		return { wishlist: [], success: false };
+	}
+};
+
+const addToWishlist = async (wishlisted_item) => {
+	const headers = { authorization: auth };
+	const product = wishlisted_item;
```

Comment:
this is probably unnecessary since you can set the parameter itself to product
`const addToWishlist = async (product) => {`

Reply:
fixed